src/Platforms/Gcc/UtestPlatform.cpp

```diff
@@ -235,7 +235,7 @@ void* (*PlatformSpecificMemset)(void*, int, size_t) = memset;
 /* MinGw GCC 5.3.0 isnan / isinf macros are not working properly, causing a conversion
  * warning / error
  */
-#ifdef __MINGW32__
+#if defined(__MINGW32__) || (defined(__GNUC__) && __GNUC__ >= 5)
```

Comment:
Based on the failure in PR #874, it appears that we definitely want to only check for the GCC version.  MinGW/GCC 4 is breaking there because it doesn't know the #pragma that this wraps.